### PR TITLE
Fix doc for NODE column

### PR DESCRIPTION
**Pull request by HackToday**
Swarm docker ps output nolonger have NODE column, So this updates the doc

Fixes: #2081
Signed-off-by: Kai Qiang Wu(Kennan) <wkqwu@cn.ibm.com>

### CODE REVIEW

**Comment by vieux** [comment]
LGTM

File: docs/scheduler/filter.md [hunk 1]
```diff
@@ -260,9 +260,9 @@ $ docker tcp://<manager_ip:manager_port> run -d --name redis7 -e affinity:image=
...
 
 $ docker tcp://<manager_ip:manager_port> ps
-CONTAINER ID        IMAGE               COMMAND             CREATED                  STATUS              PORTS                           NODE        NAMES
-87c4376856a8        redis:latest        "redis"             Less than a second ago   running                                             node-1      redis1
-1212386856a8        redis:latest        "redis"             Less than a second ago   running                                             node-1      redis2
+CONTAINER ID        IMAGE               COMMAND             CREATED                  STATUS              PORTS                           NAMES
+87c4376856a8        redis:latest        "redis"             Less than a second ago   running                                             node-1/redis1
+1212386856a8        redis:latest        "redis"             Less than a second ago   running                                             node-1/redis2
 87c4376639a8        redis:latest        "redis"             Less than a second ago   running                                             node-3      redis3
```
Comment (dongluochen) [inline docs/scheduler/filter.md]: Need update here.

File: docs/scheduler/filter.md [hunk 2]
```diff
@@ -376,9 +376,9 @@ $ docker tcp://<manager_ip:manager_port> run -d -p 80:80 nginx
 963841b138d8
 
 $ docker tcp://<manager_ip:manager_port> ps
-CONTAINER ID   IMAGE               COMMAND        PORTS                           NODE        NAMES
-f8b693db9cd6   nginx:latest        "nginx"        192.168.0.44:80->80/tcp         node-3      stoic_albattani
-963841b138d8   nginx:latest        "nginx"        192.168.0.43:80->80/tcp         node-2      dreamy_turing
+CONTAINER ID   IMAGE               COMMAND        PORTS                           NAMES
+f8b693db9cd6   nginx:latest        "nginx"        192.168.0.44:80->80/tcp         node-3/stoic_albattani
+963841b138d8   nginx:latest        "nginx"        192.168.0.43:80->80/tcp         node-2/dreamy_turing
 87c4376856a8   nginx:latest        "nginx"        192.168.0.42:80->80/tcp         node-1      prickly_engelbart
```
Comment (dongluochen) [inline docs/scheduler/filter.md]: Need update.

File: docs/scheduler/filter.md (hunk 1 above)
Comment (HackToday) [inline docs/scheduler/filter.md]: hi @dongluochen  yes, seems missed :0  Do we need another PR for this? 
Thanks

Comment (dongluochen) [inline docs/scheduler/filter.md]: yes please add another PR.
